pkg/kontainer-engine/drivers/aks/aks_driver.go

```diff
@@ -1433,11 +1307,3 @@ func (d *Driver) GetK8SCapabilities(ctx context.Context, _ *types.DriverOptions)
 	}, nil
 }
 
-func generateUniqueLogWorkspace(workspaceName string) string {
```

Comment:
There is a test using this, make sure you are running tests locally. Did this test get merged to ask-operator ?

Reply:
good catch, thank you I will move it there

Reply:
It was added in https://github.com/rancher/aks-operator/pull/23